Example/MoyaComparison/MoyaComparison/Utils/WebServiceFactory.swift

```diff
@@ -0,0 +1,49 @@
+//
+// Copyright 2017 niji
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+// http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+//
+
+import Foundation
+import NetworkStack
+import Moya
+
+struct WebServiceFactory {
```

Comment:
Given that this factory creates both WS Clients and just regular Services, maybe rename this `struct ServiceFactory` rather than `WebServiceFactory`. Then you could rename `static func clients()` into `static func wsClients()` but keep `status func services()`

Also, I see that this object only has `static` functions:

* either use an `enum` with no case rather than a `struct` (to prevent people to do `WebServiceFactory.init()` which would be pointless if there's not instance method nor properties in that object), or 
* or decide that people should create instances, so make your functions `wsClients()` and `services()` non-static, and at call site you'll have to create an instance of `ServiceFactory()` to call them… but I don't see a real interest for that solution in an Example project